com.github.jkotra.unlockr.yml

```diff
@@ -0,0 +1,36 @@
+app-id: com.github.jkotra.unlockr
+runtime : org.gnome.Platform
+runtime-version : '42'
+sdk : org.gnome.Sdk
+command: unlockr
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host
```

Comment:
that's a very broad filesystem permission

Reply:
I based my manifest on my previous app that required those, didn't notice that. sorry. fixed now.